Frontend/src/test/pages/cart/Cart.test.jsx

```
@@ -0,0 +1,148 @@
+  CardHeader: ({ children }) => <div>{children}</div>,
+  CardTitle: ({ children }) => <h2>{children}</h2>,
+}));
+vi.mock('../../../pages/cart/components/LoadingCart', () => ({ default: () => <div>LoadingCart</div> }));
+vi.mock('../../../pages/cart/components/ErrorCart', () => ({ default: ({ error }) => <div>ErrorCart: {error}</div> }));
+vi.mock('../../../pages/cart/components/EmptyCart', () => ({ default: () => <div>EmptyCart</div> }));
+vi.mock('../../../pages/cart/components/NeedHelp', () => ({ default: () => <div>NeedHelp</div> }));
+vi.mock('../../../pages/cart/components/OrderSummary', () => ({
+  default: ({ subtotal, total, handleCheckout }) => (
+    <div data-testid="order-summary">
+      Summary {subtotal}-{total}
+      <button onClick={handleCheckout}>Checkout</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)